test/Engine/FFITarget/DummyZeroTouchClass.cs

```diff
@@ -16,4 +17,16 @@ public int FunctionWithoutDescription(int a)
             return 0;
         }
     }
+
+    [InPortNames("par1", "par2")]
+    [InPortTypes("int", "double")]
+    [InPortDescriptions(typeof(Resources), "DescriptionResource1")]
+
+    [OutPortNames("out1", "out2", "out3")]
+    [OutPortTypes("1", "2", "3")]
+    [OutPortDescriptions("desc")]
+    public class DummyZeroTouchClass1
```

Comment:
It's erroneous to put this on a zero touch class - these attributes should only be applied to `NodeModel` sub classes. We should put this on a dummy class called `DummyNodeModel`. These port descriptions should generally not be applied to